src/Storefront/Page/GenericPageLoaderInterface.php

```diff
@@ -0,0 +1,11 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Storefront\Page;
+
+use Shopware\Core\System\SalesChannel\SalesChannelContext;
+use Symfony\Component\HttpFoundation\Request;
+
+interface GenericPageLoaderInterface
```

Comment:
You probably wanna use it for decoration right?

then you can annotate the interface and class implmenting it with `@Decoratable`
that way it is clear for external developers that these classes are designed to be decorated and furthermore custom phpStan rules will kick in and will ensure that these classes really are decoratable see @ `\Shopware\Core\System\Annotation\Concept\ExtensionPattern\Decoratable
`

Reply:
Hi @keulinho, thanks for your reply! I added the annotation in the classes and interfaces - I don't think there are any attributes or parameters to it?

Reply:
nope looks good :+1:

Reply:
I'm not sure about the failing tests - it doesn't look related to this PR

Reply:
the travis build is not reliable at the moment :see_no_evil: sorry!

Reply:
Thanks for the feedback!